assets/src/web-stories-block/css/views/grid.css

```diff
@@ -0,0 +1,18 @@
+/* Web Stories: Grid View */
+.web-stories-list.is-view-type-grid {
+  display: flex;
+  flex-wrap: wrap;
+
+  .web-stories-list__story-wrapper {
+    flex-basis: calc(100% - 8px);
+    margin: 4px;
+  }
+
+  @media all and (min-width: $ws-mq-medium) {
+    @for $i from 1 through 5 {
+      &.columns-#{$i} .web-stories-list__story-wrapper {
+        flex-basis: calc((100 / #{$i} * 1%) - 8px);
+      }
+    }
```

Comment:
Other than it's clever, why is this a for loop?

Reply:
Build process helps with writing less, I used this to avoid writing each columns and in case we need to add support for more columns can be done.